### DISCUSSION TITLE
Why can't get Twitter feed?

### DISCUSSION THREAD

**Discussion by vaniaxx** [DIYgod/RSSHub#14830] [Q&A] [answered]
I've been struggling with this issue for two weeks now. I'm using Zeabur.
I've put everything under **environment** in **docker-compose.yml** but it doesn't work.

    TWITTER_USERNAME: ???
    TWITTER_PASSWORD: ???
    TWITTER_AUTHENTICATION_SECRET: ???
    TWITTER_COOKIE: 'auth_token=yourtoken; ct0=yourkey'

Error Message: Twitter API is not configured

**Comment by TonyRL** [comment, marked as answer]
Since you've mentioned Zeabur please do read their docs on how to configure environment variable. Putting them in docker compose file won't work IIRC.

**Reply by vaniaxx** [reply]
Thank you so much for your reply! I was able to solve all of my problems thanks to your advice! **I succeeded in getting my Twitter feed!**

I realized that I was making a really basic mistake. All I had to do was create from the Zeabur RSSHub template and register this information in **Edit Raw Variables**. There was also a problem with the way I was writing it.

I realized this on this page.
https://zeabur.com/docs/deploy/variables

> Click the Edit as Raw button to enter the batch editing mode, which will set the service environment variables in .env file.

I think many people want to get their Twitter feed, so I would like to introduce the procedure in my native language on my blog and SNS. Once you understand these, it is very easy, but I would like to explain the basic construction part in a more beginner-friendly way.
